<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HandlerChoiceDialog</class>
 <widget class="QDialog" name="HandlerChoiceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose entity handler</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>LeechCraft has detected an entity:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="EntityLabel_">
     <property name="text">
      <string notr="true">TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="DownloadersLabel_">
     <property name="text">
      <string>It could be downloaded by:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="DownloadersLayout_"/>
   </item>
   <item>
    <widget class="QLabel" name="HandlersLabel_">
     <property name="text">
      <string>It could be handled by:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="HandlersLayout_"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Save to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="LocationsBox_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>Browse...</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HandlerChoiceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HandlerChoiceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
